### PR TITLE
Tasks 743-746 EURLEX,YELP and ARITHMETIC 

### PR DESCRIPTION
**Pull request by RaghulRajM**

### CODE REVIEW

**Comment by swarooprm** [comment]
Reviews: https://github.com/swarooprm/CSE576HW/pull/117

**Comment by swarooprm** [comment]
Fixed several things in the definitions. 

One minor concern is task 743 it says you are given articles, but it should be an article.

Another minor concern is: sometimes there is unwanted \n in data:
{
            "input": "They NEVER seem to get our \norder correct, service is crappy, food is inconsistent and has gone down hill steadily in the last 6-9 months! WILL NEVER GO THERE AGAIN!",
            "output": [
                "negative"
            ]
        }
 
See if you can get rid of those.

Everything else look good @danyaljj

**Comment by danyaljj** [comment]
@RaghulRajM could you address the above suggestions?

**Comment by RaghulRajM** [comment]
@danyaljj I'm done with changes as suggested by @swarooprm . Apologies for the delay in response.

**Comment by swarooprm** [comment]
@RaghulRajM, can you address the following also which is not addressed yet?
One minor concern is task 743 it says you are given articles, but it should be an article.

**Comment by RaghulRajM** [comment]
> @RaghulRajM, can you address the following also which is not addressed yet? One minor concern is task 743 it says you are given articles, but it should be an article.

@swarooprm I thought you changed it already. It's done now. Let me know if there's anything else

**Comment by swarooprm** [comment]
Fixed a task ordering issue in readme.

@danyaljj, looks good to me once tests pass!

**Comment by RaghulRajM** [comment]
Don't merge it yet. One of the test cases failed. Let me fix that and push again.

**Comment by danyaljj** [comment]
Side note: @RaghulRajM  Make sure that your email is set in your git environment and it is also mentioned in your GitHub profile
https://github.com/allenai/natural-instructions-expansion#my-commits-are-not-tied-to-my-github-profie-github-does-not-show-all-my-contributions

**Comment by RaghulRajM** [comment]
> Side note: @RaghulRajM Make sure that your email is set in your git environment and it is also mentioned in your GitHub profile https://github.com/allenai/natural-instructions-expansion#my-commits-are-not-tied-to-my-github-profie-github-does-not-show-all-my-contributions

Yeah I'll do that @danyaljj and I'm done with the changes too

File: tasks/README.md [hunk 1]
```diff
@@ -264,6 +264,9 @@ Name | Summary | Category
 `task261_spl_translation_es_en` | Translate Spanish questions to English while preserving named entities in the original language. | Translation
 `task262_spl_translation_ja_en` | Translate Japanese questions to English while preserving named entities in the original language. | Translation
 `task263_spl_translation_pl_en` | Translate Polish questions to English while preserving named entities in the original language. | Translation
+`task264_paper_reviews_accept_or_reject_classification` | Given a set of reviews, classify paper into accept or reject | Classification
```
Comment (danyaljj) [inline tasks/README.md:267]: good, thanks!

**Comment by danyaljj** [comment]
Looks good, thanks!

**Pull request merged**